stripe-core/src/main/java/com/stripe/android/core/networking/ApiRequest.kt

```diff
@@ -195,6 +196,11 @@ data class ApiRequest internal constructor(
 
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     companion object {
-        const val API_HOST = "https://api.stripe.com"
+        val API_TEST_HOST = AtomicReference<String?>(null)
```

Comment:
I don't love this. I was really hoping to be able to change this in our ConnectionFactory and change the URL there.

Reply:
Why does it have to be an AtomicReference?